dashboard-widgets.el

```diff
@@ -1207,6 +1216,19 @@ If not height is found on FACE or `dashboard-items-face' use `default'."
                        (dashboard-agenda--entry-timestamp (point)))))
     (format-time-string dashboard-agenda-time-string-format time)))
 
+(defun dashboard-agenda--formatted-tags ()
+  "Apply `dashboard-agenda-tags-format' to org-element tags."
+  (let ((tags (org-get-tags)))
+    (apply dashboard-agenda-tags-format tags)))
+
+(defun dashboard-agenda--show-tags (&rest tags)
+  "Show all TAGS."
+  tags)
+
+(defun dashboard-agenda--hide-tags (&rest _other_tags)
+  "Ignore TAGS and return nil"
+  nil)
```

Comment:
Maybe we can use `#ignore` function here?  Or make `dashboard-agenda-tags-format` accept `nil`? I think it will be much easier to customize it this way.

Reply:
@jcs090218 thanks for the suggestions.